.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: CI for Template
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+# Extra context is a way to override options that the user normally
+# specifies during the questions phase of the cookiecutter setup. The keys
```

Comment:
cookiecutter? I thought this was copier!

Reply:
Oh noes, my copypasta caught up with me! The shame.

Reply:
Removed the comments.